### PR TITLE
Add sort-mentions extension

### PR DESCRIPTION
**Pull request by YamenSharaf**
## Description

A very small utility extension to sort the mentioned people alphabetically in apps like Slack or Discord

Usually when I'm mentioning multiple people I don't want to imply the importance of one person over the other so I've had this utility as a script in the past, but might as well share it as an extension

## Features
- Sort usernames starting with `@` alphabetically 
- Sort channels starting with `#`
- Ability to change either of these prefixes for a more customized experience.
- Configurability to choose what happens after sorting:
  - Replace selected text vs copy to clipboard
  - Play a system sound on success or not

## Screencast

https://github.com/raycast/extensions/assets/25423453/1b684fcd-6590-4958-a7f1-6097d9222073

Screenshot 19 07 2023-11 39 28

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/sort-mentions/package.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  "author": "Yamen",
+  "categories": [
+    "Communication"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "sort-primary",
+      "title": "Sort Usernames",
+      "subtitle": "Sort usernames alphabetically",
```
Comment (pernielsentikaer) [inline extensions/sort-mentions/package.json:16]:
```suggestion
      "title": "Sort Usernames",
```

File: extensions/sort-mentions/package.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      "description": "Sort usernames alphabetically",
+      "mode": "no-view"
+    },
+    {
+      "name": "sort-secondary",
+      "title": "Sort Custom",
+      "subtitle": "Custom sort alphabetically",
```
Comment (pernielsentikaer) [inline extensions/sort-mentions/package.json:23]:
```suggestion
      "title": "Sort Channels",
```

File: extensions/sort-mentions/src/utils.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { Clipboard, Toast, getPreferenceValues, getSelectedText, showToast } from "@raycast/api";
+import { exec } from "child_process";
+
+export const handleInput = async (handle_type: Preferences["primary_handle"] | Preferences["secondary_handle"]) => {
+  const selectedText = await getSelectedText();
```
Comment (pernielsentikaer) [inline extensions/sort-mentions/src/utils.ts:5]: If not selecting any text it crashes, could you handle this gracefully? You can see an [example here](https://developers.raycast.com/api-reference/environment#example-3)

sort-mentions 2023-07-25 at 09 42 58

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by YamenSharaf** [comment]
Thank you @pernielsentikaer for taking the time to review. I've committed new changes to address the issues you mentioned.

Please feel free to take a second look and let me know if there's something I can improve 🙏🏻

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**